src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,72 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+
+@Embeddable
+public class Answers {
+
+	@OneToMany(mappedBy = "question")
+	private final List<Answer> answerList;
+
+	protected Answers() {
+		this.answerList = new ArrayList<>();
```

Comment:
```suggestion
		this(new ArrayList<>());
```

인스턴스 변수의 초기화는 주 생성자를 정하여 하나의 생성자에서 해주는 것이 좋은데, 왜 그럴지 고민해보아요 ㅎㅎ

Reply:
아하 네! 하나의 생성자에서 하는 것이 변경에 더 유연할 것 같네요. 한 곳만 변경하면 되니.